File: 2023/php/test/TestDay2A.php
<?php declare(strict_types=1);

$loader = require __DIR__ . '/../vendor/autoload.php';

use day2\a\Solution2A;
use PHPUnit\Framework\TestCase;
use utils\TestLogger;

final class TestDay2A extends TestCase
{

    public function testExample(): void
    {
        $this->assertSame(8, Solution2A::getResult("../resources/2/test_a.txt", TestLogger::createTestLogger()));
    }

    public function testResult(): void
    {
        $this->assertSame(2776, Solution2A::getResult("../resources/2/input.txt", TestLogger::createTestLogger()));
    }

}

File: 2023/java/src/test/java/org/frieder/aoc/day2/TestDay2A.java
package org.frieder.aoc.day2;

import org.frieder.aoc.day2.a.Solution2A;
import org.junit.Test;

import java.io.IOException;

import static org.junit.Assert.assertEquals;

public class TestDay2A
{
    @Test
    public void testExample() throws IOException {
        assertEquals(8, Solution2A.getResult("../resources/2/test_a.txt"));
    }

    @Test
    public void testResult() throws IOException {
        assertEquals(2776, Solution2A.getResult("../resources/2/input.txt"));
    }
}
